src/idsk/components/footer-extended/footer-extended.scss

```diff
@@ -96,80 +81,313 @@
         font-size: 17px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
     }
 
-    .label {
+    .idsk-label {
         font-style: normal;
         font-weight: normal;
         font-size: 19px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
         text-decoration: underline;
     }
 
-    .bottom-space {
+    .idsk-grid-column {
         padding-bottom: 23px;
     }
 
+    .idsk-text-h3 {
+        font-family: Source Sans Pro;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 21px;
+        line-height: 25px;
+        color: govuk-colour("black");
+        font-weight: bold;
+    }
+
+    .heading-feedback {
+        margin: 0 -15px;
+        background: govuk-colour("blue");
+    }
+
+    .feedback-text {
+        font-family: Source Sans Pro;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 12px;
+        line-height: 23px;
+        color: govuk-colour("white");
+    }
+
+    @include mq(tablet) {
+        .govuk-heading-m {
+            font-size: 1.39em;
+        }
+    }
+
+    @include mq($from: tablet) {
+        .feedback-text {
+            top: 7.49%;
+            bottom: 91.98%;
+            font-family: Source Sans Pro;
+            font-style: normal;
+            font-weight: bold;
+            font-size: 18px;
+            line-height: 23px;
+            color: govuk-colour("white");
+        }
+    }
+
+    .right-text {
+        float: right;
+    }
+
+    @include mq(tablet) {
+        .text-padding {
+            padding-left: 34px;
+        }
+    }
+
+    @include mq($from: desktop) {
+        .text-padding {
+            padding-left: 70px;
+        }
+
+        .govuk-heading-m {
+            font-size: 1.5em;
+        }
+    }
+    .feedback-container {
+        padding-top: 0;
+        padding-left: 15px;
+        overflow: hidden;
+    }
+
+    @include mq($from: desktop) {
+        .feedback-container {
+            padding-top: 15px;
+            padding-left: 25px;
+            padding-bottom: 17px;
+        }
+    }
+
+    .feedback-container.collapsed {
+        max-height: 0;
+    }
+
+    .feedback-button {
+        padding: 0 0 0 25px;
+    }
+
+    .idsk-close-button {
+        padding-top: 20px;
+    }
+
+    .help-button {
+        background: none;
+        border-style: none;
+        text-decoration: underline;
+        cursor: pointer;
+    }
+
+    .blue-border {
+        border-top: 5px solid govuk-colour("blue");
+        border-bottom: 5px solid govuk-colour("blue");
+        background-clip: border-box;
+    }
+
+    .form-text {
+        font-family: Source Sans Pro;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 19px;
+        line-height: 25px;
+        color: govuk-colour("black");
+    }
+
+    .idsk-display-none {
+        visibility: hidden;
+    }
+
+    .idsk-display- {
+        display: none;
+    }
```

Comment:
idsk-display-none nepopisuju to co robi, popisuje vlastne spravanie idsk-display-.. plus to iste co skoro vsade idsk-something je skor pre globalne classy nie specificky pre footer/feedback